bindgen/ir/context.rs

```diff
@@ -356,6 +356,11 @@ pub(crate) struct BindgenContext {
     /// This needs to be an std::HashMap because the cexpr API requires it.
     parsed_macros: StdHashMap<Vec<u8>, cexpr::expr::EvalResult>,
 
+    /// A map with all include locations.
+    ///
+    /// This is needed so that items are created in the order they are defined in.
```

Comment:
could you either document to which filename each element of the tuple corresponds to or add a dedicated `struct` with named fields for this?